### DISCUSSION TITLE
Consider adding more examples, like react-flow

### DISCUSSION THREAD

**Discussion by laterdayi** [bcakmakoglu/vue-flow#868] [Ideas]
Consider adding more examples, like react-flow

like Position auxiliary line， collision detection ，ntersections, edges, automatic updates, custom nodes, etc....

**Comment by laterdayi** [comment]
It is excellent, but relative to antdX6, react flow, do you need more documentation and demo

**Reply by bcakmakoglu** [reply]
Seeing as I do this all by myself, this is the best I can deliver while also working a 9-5 job 5 days a week.
Feel free to add more examples and open a PR for it 😅 I’m always open for contributions 
Though I do plan a relaunch of the documentation and will add more examples it just takes time as, again, I do this by myself and not with a team of multiple people 🥲

**Reply by bcakmakoglu** [reply]
Also a lot of the examples you listed *are* available on the docs or in the repo in the vite examples.
Auxiliary lines I Won‘t do as it’s a pro example (paid) by react flow and I’m not going to subvert their payment scheme by making it available for free

**Comment by laterdayi** [comment]
Thank you very much for your contributions,` vue-flow `is excellent, and during this discussion I saw an invitation from members of the Apache team to work with them to make ` vue-flow ` even more powerful

**Comment by laterdayi** [comment]
https://github.com/bcakmakoglu/vue-flow/discussions/427

**Comment by laterdayi** [comment]
I'm not sure if there's more to follow. Being a great plugin, it should be maintained by more people, and it should be in a longer state

**Comment by laterdayi** [comment]
Are there more planned start and end points for document refactoring? Maybe consider vitepress for an easier ride

**Reply by bcakmakoglu** [reply]
It already uses Vitepress 😅
